<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RD Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="opaqueResize">
            <bool>true</bool>
           </property>
           <property name="childrenCollapsible">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="plotsFrame" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Table</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget{
background-color: transparent;
gridline-color: grey;
}
QTableWidget::item {
background-color: white;
}
QTableWidget::item:selected {
background-color: grey;
}

QHeaderView {
background-color: transparent;
gridline-color: grey;
}
QHeaderView::section{
border-top: 1px solid;
border-top-color: grey;
border-left:1px solid;
border-left-color: grey;
border-bottom: 1px solid;
border-bottom-color: grey;
border-right: 1px solid;
border-right-color: grey;
padding: 4px;
}
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="plotsettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>329</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Plotsettings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeView" name="variableTreeView">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Bjontegaard Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_interp">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_rate_psnr"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_bdplot">
       <property name="text">
        <string>Plot Bjontegaard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Confidence Interval Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_ci">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1253</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <widget class="QMenu" name="menuRecent_files">
     <property name="title">
      <string>Recent files</string>
     </property>
    </widget>
    <addaction name="actionOpen_File"/>
    <addaction name="menuRecent_files"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Directory"/>
    <addaction name="actionOpen_Directory_List"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Table"/>
    <addaction name="actionExport_Figure_as_Tikzpicture"/>
    <addaction name="actionExport_TableWidget"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Data"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionHide_Sequence"/>
    <addaction name="actionHide_PlotSettings"/>
    <addaction name="actionHide_Status"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRemove_items"/>
    <addaction name="actionReload_files"/>
    <addaction name="separator"/>
    <addaction name="actionGenerate_curve"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="sequenceWidget">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Sequences</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="SimDataItemTreeView" name="simDataItemTreeView">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DropOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="statusWidget">
   <property name="windowTitle">
    <string>Plotted files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PlottedFilesListView" name="simDataItemListView">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="curveWidget">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Generated curves</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CurveView" name="curveListView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>&amp;Open File (RD-Data or Log)</string>
   </property>
  </action>
  <action name="actionOpen_Directory">
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Plot">
   <property name="text">
    <string>&amp;Save Plot</string>
   </property>
  </action>
  <action name="actionHide_Sequence">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hide &amp;Sequences</string>
   </property>
  </action>
  <action name="actionHide_PlotSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide &amp;Plotsettings</string>
   </property>
  </action>
  <action name="actionSave_all_Plots">
   <property name="text">
    <string>Save all &amp;Plots</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpen_Sequence">
   <property name="text">
    <string>Ope&amp;n Sequence</string>
   </property>
  </action>
  <action name="actionHide_Status">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hide Status</string>
   </property>
  </action>
  <action name="actionSave_Table">
   <property name="text">
    <string>&amp;Save Table</string>
   </property>
  </action>
  <action name="actionExport_TableWidget">
   <property name="text">
    <string>&amp;Export table view ... as</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Sa&amp;ve selected data ... as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Directory_List">
   <property name="text">
    <string>O&amp;pen Directory List</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExport_Figure_as_Tikzpicture">
   <property name="text">
    <string>Export Figure as Tikzpicture</string>
   </property>
  </action>
  <action name="actionPlaceholder">
   <property name="text">
    <string>Placeholder</string>
   </property>
  </action>
  <action name="actionRemove_items">
   <property name="text">
    <string>Remove items</string>
   </property>
  </action>
  <action name="actionReload_files">
   <property name="text">
    <string>Reload files</string>
   </property>
  </action>
  <action name="actionGenerate_curve">
   <property name="text">
    <string>Generate curve</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimDataItemTreeView</class>
   <extends>QTreeView</extends>
   <header>rdplot.view</header>
  </customwidget>
  <customwidget>
   <class>PlottedFilesListView</class>
   <extends>QListView</extends>
   <header>rdplot.view</header>
  </customwidget>
  <customwidget>
   <class>CurveView</class>
   <extends>QListView</extends>
   <header>rdplot.view</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
